docs/get-started/quixtour/process-threshold.md

```diff
@@ -13,35 +13,39 @@ To create the threshold detection transform:
 5. Name the transform "CPU Threshold".
 6. Select the input topic `cpu-load`.
 7. For the output topic, add a new topic called `cpu-spike`.
-8. In the project view click on `main.py` to edit it.
+8. In the application view, click on `main.py` to edit it.
 9. Replace all the code in `main.py` with the following:
 
     ``` python
     from quixstreams import Application
     import os
 
-    
     # Create an Application
     # It will get the SDK token from environment variables to connect to Quix Kafka
     app = Application.Quix(consumer_group="threshold-transform", auto_offset_reset="latest")
 
     # Define input and output topics
-    input_topic = app.topic(os.environ["input"], value_deserializer='json')
-    output_topic = app.topic(os.environ["output"], value_serializer='json')
+    input_topic = app.topic(os.environ["input"])
+    output_topic = app.topic(os.environ["output"])
 
     # Create a StreamingDataFrame to process data
     sdf = app.dataframe(input_topic)
 
-    # Filter messages with "cpu_load" > 20
-    sdf = sdf[sdf["cpu_load"] > 20]
+    # Filter in all rows where CPU load is over 20.
+    sdf = sdf.filter(lambda row: row["cpu_load"] > 20)
 
-    # Print filtered messages to the console
-    sdf = sdf.update(lambda row: print(row))
```

Comment:
I would keep update, maybe after the last apply so they see it is working.

Reply:
update added after apply